### PR TITLE
Bind additional contexts to flush requests

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This binds new context vars to our logs on flush requests. New values are exercise id, schema name and schema url where applicable. 

### How to review
Two new unit tests added to test both logs with and without `schema_url` present.

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by petechd** [comment]
[Bind additional contexts to flush requests (1 SP)](https://trello.com/c/xWkIgZD3/5678-bind-additional-contexts-to-flush-requests-1-sp)

File: app/routes/flush.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ def flush_data() -> Response:
 
         if metadata := get_metadata(user):
             contextvars.bind_contextvars(tx_id=metadata.tx_id)
+            contextvars.bind_contextvars(ce_id=metadata.collection_exercise_sid)
```
Comment (MebinAbraham) [inline app/routes/flush.py:59]: Can be in a single call 
```py
contextvars.bind_contextvars(
    tx_id=metadata.tx_id,
    ce_id=metadata.collection_exercise_sid,
)
```

**Review by berroar — APPROVED** [review]
Tested, logs updated, LGTM 👍

**Review by katie-gardner — APPROVED** [review]
New tests cover one of the optional variables being there and not and make sense to me, consistent with other similar log tests. Functional change looks good 👍

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**